osu.Game.Rulesets.Catch/Skinning/Default/PulpFormation.cs

```diff
@@ -34,11 +33,23 @@ protected PulpFormation()
             distance * MathF.Sin(angle * MathF.PI / 180),
             distance * MathF.Cos(angle * MathF.PI / 180));
 
-        [BackgroundDependencyLoader]
-        private void load(DrawableHitObject drawableObject)
+        protected void Clear()
+        {
+            for (; numPulps > 0; numPulps--)
+                InternalChildren[numPulps - 1].Alpha = 0;
+        }
+
+        protected void Add(Vector2 position, Vector2 size)
```

Comment:
Would rename to `AddPulp` to convey intention better (looks really weird at call sites, like the generic `Drawable.Add()` but without a drawable).